src/components/SearchGridFilter.vue

```diff
@@ -92,41 +169,108 @@ export default {
         shouldNavigate: true,
       });
     },
-    onUpdateSearchByCreator() {
-      this.$store.dispatch(TOGGLE_FILTER, {
-        filterType: 'searchBy',
+    onClearFilters() {
+      this.$store.commit(CLEAR_FILTERS, {
         isCollectionsPage: this.$props.isCollectionsPage,
         provider: this.$props.provider,
         shouldNavigate: true,
       });
     },
-    onClearFilters() {
-      this.$store.commit(CLEAR_FILTERS, {
+    onUpdateSearchByCreator() {
+      this.$store.dispatch(TOGGLE_FILTER, {
+        filterType: 'searchBy',
         isCollectionsPage: this.$props.isCollectionsPage,
         provider: this.$props.provider,
         shouldNavigate: true,
       });
     },
+    close() {
+      this.$store.commit(
+        SET_FILTER_IS_VISIBLE,
+        { isFilterVisible: !this.isFilterVisible },
```

Comment:
This should force it to `false`, since it's closing the modal

Reply:
Done.